app/main/views/frameworks.py

```diff
@@ -95,6 +95,7 @@ def update_framework(framework_slug):
         'status': 'status',
         'clarificationQuestionsOpen': 'clarification_questions_open',
         'frameworkAgreementDetails': 'framework_agreement_details',
+        'applicationCloseDate': 'application_close_date',
```

Comment:
Do we need to bother validating that the supplied date is in the right format anywhere before saving? I can't see a schema for a framework object, only the `frameworkAgreementDetails`.

Reply:
It saves as a datetime field rather than a string.  You can post various formats that work (e.g. `2017-04-11 16:00`.  And if you post something that doesn't work the DB will choke on it, which I think we can live with as a validation.

Reply:
Fair enough 👍